integration_tests/CMakeLists.txt

```diff
@@ -171,6 +171,7 @@ RUN(NAME elemental_01        LABELS cpython llvm)
 RUN(NAME elemental_02        LABELS cpython llvm)
 RUN(NAME elemental_03        LABELS cpython llvm)
 RUN(NAME elemental_04        LABELS cpython llvm)
+RUN(NAME elemental_05        LABELS cpython llvm)
```

Comment:
Please make it `elemental_06`. There is another PR which is adding `sinh` and `cosh` in `elemental_05`.

Reply:
Thanks for quick reply.